src/server/server.rs

```diff
@@ -258,7 +258,7 @@ impl<T: RaftStoreRouter<RocksEngine> + Unpin, S: StoreAddrResolver + 'static> Se
             server.shutdown();
         }
         if let Some(pool) = self.stats_pool.take() {
-            let _ = pool.shutdown_timeout(Duration::from_secs(60));
```

Comment:
why we have this before

Reply:
It's a graceful style for shutting down a pool. However the only one future which is spawned in the pool is an infinate one. So it will block 60s.